atlasdb-cassandra/src/test/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfigsTest.java

```diff
@@ -118,4 +119,40 @@ public void canParseRuntimeDeprecatedConfigType() throws IOException {
                 AtlasDbConfigs.OBJECT_MAPPER.readValue("type: CassandraKeyValueServiceRuntimeConfig",
                         CassandraKeyValueServiceRuntimeConfig.class);
     }
+
+    @Test
+    public void canDeserializeDefaultDisabled() throws IOException, URISyntaxException {
+        CassandraKeyValueServiceConfig testConfig = ImmutableCassandraKeyValueServiceConfig.builder()
```

Comment:
from above, I think have localised tests where you deserialise to `CassandraServersConfig` or whatever it's called, and then you can amend one of the `CassandraKeyValueServiceConfig` to have a variation of the three configs.

Reply:
Not sure if I covered everything you wanted, please double check.